client/http-client/Cargo.toml

```diff
@@ -28,8 +28,11 @@ jsonrpsee-test-utils = { path = "../../test-utils" }
 tokio = { version = "1.16", features = ["net", "rt-multi-thread", "macros"] }
 
 [features]
-default = ["tls"]
-tls = ["hyper-rustls/webpki-tokio"]
+default = ["native-tls"]
+native-tls = ["hyper-rustls/native-tokio", "__tls"]
+webpki-tls = ["hyper-rustls/webpki-tokio", "__tls"]
+
+__tls = ["hyper-rustls"]
```

Comment:
I'm happy with this for clarity, but just noting that as it stands, `hyper-rustls`will automatically be bought in when `native-tls` or `webpki-tls` is enabled without needing it :)